aws_jupyter_proxy/awsconfig.py

```diff
@@ -0,0 +1,19 @@
+from notebook.base.handlers import IPythonHandler, utcnow
+import json
+from botocore.session import get_session
+
+class AwsConfigHandler(IPythonHandler):
+    def get(self):
+        response = {}
+        self.set_header("Content-Type", "application/json")
+
+        if self.request.uri == "/awsproxy/awsconfig":
```

Comment:
Is this condition still required if the `/awsproxy/awsconfig` path is being mapped to this handler in `handlers.py` ?